apps/settings/js/panels/wifi_manage_networks/panel.js

```diff
@@ -43,7 +43,10 @@ define(function(require) {
                 value.identity,
                 value.eap
               );
-              WifiContext.associateNetwork(network, function() {
+              WifiContext.associateNetwork(network, function(error) {
+                if (error) {
+                  DialogService.alert(error);
```

Comment:
It seems not a good idea to display the error message directly to users using dialog service. Is it possible to map it to an user-friendly message?